lib/DependencyScan/ScanDependencies.cpp

```diff
@@ -90,16 +90,15 @@ class ExplicitModuleDependencyResolver {
     commandline = resolvingDepInfo.getCommandline();
   }
 
-  llvm::Error
-  resolve(const std::set<ModuleDependencyID> &dependencies,
-          std::optional<std::set<ModuleDependencyID>> bridgingHeaderDeps) {
+  bool resolve(const std::set<ModuleDependencyID> &dependencies,
```

Comment:
Is there an established convention for `bool` and errors in the swift codebase? If not I would add a doc comment to any of these functions indicate which return value is the error path.

Reply:
In swift codebase, `true` means error but I will add a comment.